main.go

```diff
@@ -83,12 +91,27 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
+
+	scheme = controllers.GetKubeClientSchemes(scheme)
```

Comment:
This can be moved to be under `else` in case `if agentsNamespace != "" {` (line 75) is true.
Otherwise we don't really need another manager

Reply:
Sure, I will handle that